### PR TITLE
Work around a locale issue in Ormolu

### PR DESCRIPTION
**Pull request by arianvp**
Otherwise ormolu doesn't work

### CODE REVIEW

**Review by mdimjasevic — APPROVED** [review]

**Comment by mdimjasevic** [comment]
@arianvp and @pcapriotti , I just updated the PR to use the `LOCALE_ARCHIVE` variable instead, i.e., this should be a proper workaround for the locale issue.

File: shell.nix [hunk 1]
```diff
@@ -58,6 +58,7 @@ let
   };
 in pkgs.mkShell {
   name = "shell";
+  LOCAL_ARCHIVE=${glibcLocales}/lib/locale/locale-archive; # works around https://github.com/tweag/ormolu/issues/38
```
Comment (akshaymankar) [inline shell.nix:61]: Shouldn't this be `pkgs.glibcLocales`?

Comment (mdimjasevic) [reply shell.nix:61]: I am an absolute beginner at Nix. Should I replace "glibcLocales" with "pkgs.glibcLocales"?

**Review by pcapriotti — CHANGES_REQUESTED** [review]
This doesn't look right. `LOCALE_ARCHIVE` should be an environment variable, not an attribute.

**Comment by arianvp** [comment]
@pcapriotti attributes of a derivation are exposed as environment variables.

File: shell.nix (hunk 1 above)
Comment (pcapriotti) [reply shell.nix:61]: You need to wrap it in quotes and qualify `glibcLocales` with `pkgs`.

File: shell.nix [hunk 2]
```diff
@@ -58,6 +58,7 @@ let
   };
 in pkgs.mkShell {
   name = "shell";
+  LOCAL_ARCHIVE="${pkgs.glibcLocales}/lib/locale/locale-archive"; # works around https://github.com/tweag/ormolu/issues/38
```
Comment (pcapriotti) [inline shell.nix:61]:
```suggestion
  LOCALE_ARCHIVE="${pkgs.glibcLocales}/lib/locale/locale-archive"; # works around https://github.com/tweag/ormolu/issues/38
```

**Review by pcapriotti — CHANGES_REQUESTED** [review]
I think it should be `LOCALE_ARCHIVE`, not `LOCAL_ARCHIVE`, right? Have you tried to see if it fixes your issue?

**Review by pcapriotti — APPROVED** [review]

**Pull request merged**